### PR TITLE
fix: change My Collection Insights Average Price copy to Median Price 

### PR DESCRIPTION
**Pull request by olerichter00**
### Description

This PR changes the copy in the My Collection insights from "Average Price" to "Median Price" (see screenshot) and renames the corresponding components and routes.

This PR doesn't change the data that is fetched. This will happen in a follow-up PR.

| | |
| --- | --- |
| Simulator Screen Shot - iPad Air (4th generation) - 2022-08-09 at 12 11 00 | Simulator Screen Shot - iPad Air (4th generation) - 2022-08-09 at 12 11 05 |

This PR resolves [CX-2688]

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- My Collection Insights Average Auction Price -> Median Auction Price - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2688]: https://artsyproduct.atlassian.net/browse/CX-2688?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyCollection/Screens/Insights/MedianAuctionPriceRail.tsx [hunk 1]
```diff
@@ -32,12 +36,16 @@ export const AverageAuctionPriceRail: React.FC<AverageAuctionPriceRailProps> = (
       <Flex mx={2}>
         <SectionTitle
           capitalized={false}
-          title="Average Auction Price in the last 3 years"
+          title={
+            enableMyCollectionInsightsMedianPrice
+              ? "Median Auction Price in the last 3 years"
+              : "Average Auction Price in the last 3 years"
+          }
```
Comment (olerichter00) [inline src/app/Scenes/MyCollection/Screens/Insights/MedianAuctionPriceRail.tsx:43]: Copy change 1.

File: src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceAtAuction.tsx [hunk 2]
```diff
@@ -29,8 +34,8 @@ const AverageSalePriceAtAuctionScreen: React.FC<AverageSalePriceAtAuctionProps>
 
   return (
     <Flex mx={2} pt={6}>
-      <Text variant="lg" mb={0.5} testID="Average_Auction_Price_title">
-        Average Auction Price
+      <Text variant="lg" mb={0.5} testID="Median_Auction_Price_title">
+        {enableMyCollectionInsightsMedianPrice ? "Median Auction Price" : "Average Auction Price"}
```
Comment (olerichter00) [inline src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceAtAuction.tsx:38]: Copy change 2.

File: src/app/navigation/routes.ts [hunk 3]
```diff
@@ -153,7 +153,7 @@ function getDomainMap(): Record<string, RouteMatcher[] | null> {
     addRoute("/viewing-rooms", "ViewingRooms"),
     addRoute("/auction-results-for-artists-you-follow", "AuctionResultsForArtistsYouFollow"),
     addRoute("/auction-results-for-artists-you-collect", "AuctionResultsForArtistsYouCollect"),
-    addRoute("/my-collection/average-sale-price-at-auction/:artistID", "AverageSalePriceAtAuction"),
+    addRoute("/my-collection/median-sale-price-at-auction/:artistID", "MedianSalePriceAtAuction"),
```
Comment (MounirDhahri) [inline src/app/navigation/routes.ts:156]: **praise:** 🙌

**Review by MrSltun — APPROVED** [review]
🌟

**Pull request merged**